controller_report_xls/controllers/main.py

```diff
@@ -1,27 +1,4 @@
 # -*- coding: utf-8 -*-
-###########################################################################
```

Comment:
@moylop260 I think we could create an issue in the main vauxoo odoo repositories to set update python files with the short license format and also update the README.rst/index.html file to add the developer/planner/auditor of the module (extracted from the old license) to the Contributors [section](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#contributors) in the module description. Are you agree?